src/types/config.ts

```diff
@@ -35,4 +67,7 @@ export type JiterConfig = {
   signingSecret: string;
 } & JiterConfigOptions;
 
-export type JiterConfigInstance = Required<JiterConfig & OverrideJiterConfigOptions>;
+export type JiterConfigInstance = Omit<
+  Required<JiterConfig & OverrideJiterConfigOptions>,
+  'encryption'
+> & { encryption: EncryptionOptions<Buffer> | null };
```

Comment:
Why did the type change to add on a `Buffer`?

Reply:
Internally I need the keys as buffers but the user passes them in as strings.